addOns/jwt/src/main/resources/org/zaproxy/zap/extension/jwt/resources/Messages.properties

```diff
@@ -0,0 +1,78 @@
+The SameSite attribute is an effective counter measure to cross-site request forgery, cross-site script inclusion, and timing attacks.
+jwt.scanner.client.vulnerability.cookiesamesite.soln=Ensure that the SameSite attribute is set to either 'lax' or ideally 'strict' for all cookies.
+jwt.scanner.client.vulnerability.cookiesamesite.refs=https://tools.ietf.org/html/draft-ietf-httpbis-cookie-same-site
+
+jwt.scanner.client.vulnerability.cookieprefix.name=Cookie containing JWT token is without __Secure- or __Host- prefixes.
+jwt.scanner.client.vulnerability.cookieprefix.desc=A cookie has been set without the  __Secure- or __Host- prefixes, which means that the cookie can be added using Set-Cookie header via unencrypted connections or via malicious subdomains.
+jwt.scanner.client.vulnerability.cookieprefix.soln=Whenever a cookie contains sensitive information or is a session token, ensure hardening the cookie using cookie prefixes.
+jwt.scanner.client.vulnerability.cookieprefix.refs=https://googlechrome.github.io/samples/cookie-prefixes/
+
+jwt.scanner.client.vulnerability.urlparam.name=JWT token is in URL
+jwt.scanner.client.vulnerability.urlparam.desc=The request contains JWT token which is leaked in the URL. This can violate PCI and most organizational compliance policies. You can configure the list of strings for this check to add or remove values specific to your environment.
+jwt.scanner.client.vulnerability.urlparam.soln=Do not pass sensitive information in URIs.
+jwt.scanner.client.vulnerability.urlparam.refs=https://cheatsheetseries.owasp.org/cheatsheets/JSON_Web_Token_Cheat_Sheet_for_Java.html
+
+jwt.scanner.client.vulnerability.formparam.name=JWT token is in Form.
+jwt.scanner.client.vulnerability.formparam.desc=The request contains JWT token in Form param, as this is not part of cookies so assumption is that token is either present in Local Storage/Session Storage or hidden form field. In case if XSS is there then this way of storing parameter can be dangerous.
+jwt.scanner.client.vulnerability.formparam.soln=Ensure that XSS is not there in the application or Use Finger printing technique as mentioned in https://github.com/SasanLabs/JWTExtension#fingerprinting-jwt-token
+jwt.scanner.client.vulnerability.formparam.refs=https://cheatsheetseries.owasp.org/cheatsheets/JSON_Web_Token_Cheat_Sheet_for_Java.html
+
+
+# Server side JWT token vulnerabilities
+jwt.scanner.server.vulnerability.headerAttack.noneAlgorithm.name=none hashing algorithm attack
+jwt.scanner.server.vulnerability.headerAttack.noneAlgorithm.desc=JWT library accepts none hashing algorithm. none hashing algorithm is used by the JWT in case the integrity of token is already verified.so an attacker can alter the token claims and token will be trusted by the application.
+jwt.scanner.server.vulnerability.headerAttack.noneAlgorithm.soln=Not allowing none hashing algorithm.
```

Comment:
```suggestion
jwt.scanner.server.vulnerability.headerAttack.noneAlgorithm.soln=Enforce use of a real hashing algorithm.
```

Reply:
actually none has a special meaning for JWT, not sure mentioning real might make them think of other types of hashing algorithms as there are many.

Reply:
> Enforce the use of a hashing algorithm other than ‘none’.

?

Reply:
i think, it should be something like 'none' hashing algorithm should not be allowed  or Restrict the usage of 'none' hashing algorithm ?